### PR TITLE
fix shifting of headers using position:sticky

### LINKED ISSUE #16086: "Browse" headings shift when scrolling
When scrolling down on a Browse page, the header shifts. This is particularly noticeable when the first couple of columns are quite wide, as you'll see.

This is tested using current master and 5.0.2, I have not tested 4.9. Both are using PHP 7.4.

Before scrolling:
image

After scrolling down:
image

----

Before scrolling:
image

After scrolling:
image

### PR DESCRIPTION
**Pull request by rajat315315**
Signed-off-by: Rajat Jain <rajatjain.ix@gmail.com>

### Description
Used `position:sticky` & `top:60px` as described in #15806 and removed extra table `stickycolumns`.

Please describe your pull request.

Fixes #16086

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Review by MauricioFauth — COMMENTED** [review]
:+1:  This works really well.

But looks like it's not [supported by IE](https://caniuse.com/#feat=css-sticky) (phpMyAdmin [supports](https://github.com/phpmyadmin/phpmyadmin/blob/master/.browserslistrc) IE >= 10).
Is there some polyfill for that feature?

**Comment by rajat315315** [comment]
https://github.com/wilddeer/stickyfill

File: js/stickyfill.min.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/*!
+  * Stickyfill – `position: sticky` polyfill
```
Comment (williamdes) [inline js/stickyfill.min.js:2]: Could you move this file into js/vendor?

File: libraries/classes/Display/Results.php [hunk 2]
```diff
@@ -1055,6 +1055,7 @@ private function _getTableHeadersForColumns(
     ) {
         $html = '';
 
+
```
Comment (williamdes) [inline libraries/classes/Display/Results.php:1058]:
```suggestion
```

File: js/stickyfill.min.js (hunk 1 above)
Comment (rajat315315) [reply js/stickyfill.min.js]: Sure.

**Review by williamdes — APPROVED** [review]
LGTM

**Comment by williamdes** [comment]
Thank you for your work!
Could you push a commit to fix https://travis-ci.org/github/phpmyadmin/phpmyadmin/jobs/712098291#L408
If you do feel free to squash all your commits in one :)

**Comment by rajat315315** [comment]
> Could you push a commit to fix https://travis-ci.org/github/phpmyadmin/phpmyadmin/jobs/712098291#L408

Sorry, Couldn't understand the error on L408?

> If you do feel free to squash all your commits in one :)

Done!!

**Comment by rajat315315** [comment]
Hello, updates??

**Comment by williamdes** [comment]
Hello @rajat315315 
Sorry that I forgot to reply, surely read the notification on my mobile ..

> Sorry, Couldn't understand the error on L408?

```
/home/travis/build/phpmyadmin/phpmyadmin/js/sql.js
    17:5  error  'prevScrollX' is assigned a value but never used  no-unused-vars
  1040:1  error  Trailing spaces not allowed                       no-trailing-spaces
  1042:1  error  'Stickyfill' is not defined                       no-undef
```

You can run `yarn run js-lint --fix` and manually fix the reported errors left :)

**Comment by rajat315315** [comment]
I'm kind of stuck at importing `stickyfill.min.js` polyfill in `sql.js`.
If I use `stickyfill.min.js` as a `module`, I can import it from `sql.js` but yarn complains:
`Parsing error: The keyword 'import' is reserved` (I guess because of eslint specifications.)

I don't think changing specifications is a good idea. Any work around?

**Comment by williamdes** [comment]
> import

In `phpmyadmin` the `import` keyword has no sense and should not be used for now, instead you will need to use `addScriptFiles` that you can find in the codebase on `master` branch or `addScript` on `QA_5_0`

**Review by MauricioFauth — APPROVED** [review]

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**